### DISCUSSION TITLE
Does it support reading and modifying the value of the enhancement attribute, I've tried a lot but I can't read it.

### DISCUSSION THREAD

**Discussion by xbdr419** [mozman/ezdxf#964] [Q&A]
![微信图片_20231117114935](https://github.com/mozman/ezdxf/assets/19528714/cfc8cfb4-f6af-4a21-bd52-1d0560febc67)

As shown in the picture, the value of the enhancement attribute is displayed directly, I would like to be able to read and modify it, I don't know if it is supported, thanks!

**Comment by mozman** [comment]
I don't know what an "enhancement" (damn google translate!) attribute is, and without the test data and your code I can't help - read the issue template I provided!

**Comment by xbdr419** [comment]
[file.zip](https://github.com/mozman/ezdxf/files/13388203/file.zip)

```Python
f=r'D:\cad\92ffc57198eb44bc9b330b2b0682b7f1.dxf'
doc = ezdxf.readfile(f)

block = doc.blocks.get('EPLFRAME')
for e in block.entity_space.entities:
    if 'prompt' in e.dxfattribs() and e.dxfattribs()['prompt'] =='项目描述':
        print(s.dxfattribs())
```
Here's the source file, and the demo.
I printed one of them, and I can only get the following information,
```Python
{
    'handle': '1E4', 
    'owner': '1D7', 
    'layer': 'EPLAN451', 
    'insert': Vec3(101.0, 9.0, 0.0), 
    'height': 2.504999999999999, 'text': '14_10011', 
    'style': '宋体', 'prompt': '项目描述', 
    'tag': '14_10011', 
    'flags': 0, 
    'lock_position': 0
}
```

**Comment by mozman** [comment]
If your want to access the attached `ATTRIB` entities, read the docs for the [INSERT](https://ezdxf.mozman.at/docs/blocks/insert.html) entity and the [tutorial](https://ezdxf.mozman.at/docs/tutorials/blocks.html#tut-blocks)
